tests/providers/microsoft/azure/hooks/test_azure_fileshare.py

```diff
@@ -41,7 +41,7 @@
 
 
 class TestAzureFileshareHook:
-    def setup(self):
```

Comment:
Also find this typo, just reminder for myself: "check other tests"

Reply:
No idea but seems like `setup` method works as `setup_method` even if it not listed in https://docs.pytest.org/en/6.2.x/xunit_setup.html ... or i miss something